CMakeLists.txt

```diff
@@ -131,12 +131,8 @@ message(STATUS "OpenSSL: Version ${OPENSSL_VERSION}")
 message(STATUS "OpenSSL: include dir at ${OPENSSL_INCLUDE_DIR}")
 message(STATUS "OpenSSL: libraries at ${OPENSSL_LIBRARIES} ${OPENSSL_SSL_LIBRARIES}")
 
-# Zbar (for QR scanner)
 if(WITH_SCANNER)
     add_definitions(-DWITH_SCANNER)
```

Comment:
What’s the point of this?

Reply:
To optionally support building GUI without scanner support. Windows docker doesn't support it yet.